lib/phoenix_live_view/channel.ex

```diff
@@ -1522,9 +1528,12 @@ defmodule Phoenix.LiveView.Channel do
 
     if live_reload_config[:notify][:live_view] do
       state.socket.endpoint.subscribe("live_view")
+      reloader = live_reload_config[:reloader] || (&Phoenix.CodeReloader.reload/1)
```

Comment:
If this is coming from `config/` files, it is slightly ackward to ask it to be a function. Usually MFArgs are used (or a module). Thoughts?